tcp_test.go

```diff
@@ -35,7 +35,7 @@ func TestTCPConnection(t *testing.T) {
 	go func() {
 		conn, err := ln.Accept()
 		if err != nil {
-			t.Fatalf("Error accepting on socket: %s", err)
+			panic(fmt.Sprintf("Error accepting on socket: %s", err))
```

Comment:
I sometimes make a `panicf` helper for these cases, but it isn't super necessary for just a handful.